themes/default/content/docs/get-started/install/_index.md

```diff
@@ -143,15 +161,15 @@ This will install the `pulumi.exe` CLI to `%USERPROFILE%\.pulumi\bin` and add it
 
 ### Manual Installation
 
-Alternatively, you can install Pulumi manually.
+Alternatively, you can install Pulumi manually using binaries built for Windows x64.
 
 <!-- markdownlint-disable url -->
-1. Download [Pulumi {{< latest-version >}} for Windows x64](https://get.pulumi.com/releases/sdk/pulumi-v{{< latest-version >}}-windows-x64.zip). For prior versions and release notes, see the [Available Versions]({{< relref "/docs/get-started/install/versions" >}}) page.
+1. Download [Pulumi {{< latest-version >}} binaries for Windows x64](https://get.pulumi.com/releases/sdk/pulumi-v{{< latest-version >}}-windows-x64.zip). For prior versions and release notes, see the [Available Versions]({{< relref "/docs/get-started/install/versions" >}}) page.
 <!-- markdownlint-enable url -->
 
-1. Unzip the file and extract the contents to a folder such as `C:\pulumi`.
+2. Unzip the file and extract the contents to a folder such as `C:\pulumi`.
 
-1. Add `C:\pulumi\bin` to your path via **System Properties** -> **Advanced** -> **Environment Variables** -> **User Variables** -> **Path** -> **Edit**.
+3. Add `C:\pulumi\bin` to your path via **System Properties** -> **Advanced** -> **Environment Variables** -> **User Variables** -> **Path** -> **Edit**.
```

Comment:
these numbers will automatically change to 1, 2, 3 form 1. 1. as part of rendering. and leaving it as 1, 1, etc can make it easier to add future items without adjusting all the following number.

Reply:
I looked at the live version at https://www.pulumi.com/docs/get-started/install/ and there the numbers don't look like they are ordered 1, 2, 3. Instead they are numbered 1, 1, 2 🤔

Reply:
ah yah the markdown lint comments messes with that functionality, ignore me then!